integration-cli/docker_cli_run_test.go

```diff
@@ -1555,19 +1692,26 @@ func (s *DockerSuite) TestRunWithBadDevice(c *check.C) {
 }
 
 func (s *DockerSuite) TestRunEntrypoint(c *check.C) {
-	testRequires(c, DaemonIsLinux)
 	name := "entrypoint"
-	out, _ := dockerCmd(c, "run", "--name", name, "--entrypoint", "/bin/echo", "busybox", "-n", "foobar")
 
-	expected := "foobar"
+	// Note Windows does not have an echo.exe built in.
+	var out, expected string
+	if daemonPlatform == "windows" {
+		out, _ = dockerCmd(c, "run", "--name", name, "--entrypoint", "cmd /s /c echo", "busybox", "foobar")
+		expected = "foobar\r\n"
```

Comment:
@jhowardmsft really nbd, it's just out of curiosity, isn't there an `echo -n` equivalent in windows so that expected can be `foobar` for both?